pandas/tests/io/excel/test_readers.py

```diff
@@ -84,6 +84,7 @@ def cd_and_set_engine(self, engine, datapath, monkeypatch, read_ext):
         monkeypatch.chdir(datapath("io", "data", "excel"))
         monkeypatch.setattr(pd, "read_excel", func)
 
+    @td.check_file_leaks
```

Comment:
Is it possible to make this an autouse fixtures instead of decorating every test?

Reply:
i think so, yes

Reply:
+/- 0 . Looks like this version would have been release in Aug 2018, so less than a year and a half